apis/v1beta1/vspherevm_types.go

```diff
@@ -51,6 +59,32 @@ type VSphereVMSpec struct {
 	// this CRD as unstructured data.
 	// +optional
 	BiosUUID string `json:"biosUUID,omitempty"`
+
+	// PowerOffMode describes the desired behavior when powering off a VM.
```

Comment:
I'm a bit hesitated to do so because the naming of `VirtualMachineCloneSpec`.
It does not seem like the same semantic to suggest that `powerOffMode` should be part of the cloning process/config.

Reply:
cc @yastij :-)

Reply:
Not sure if it's worth it. quoting go proverbs "sometimes a little bit of copying is better than a small dependency".

Except we have a good idea for a struct name that makes sense